operators/jaeger/1.41.0/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,592 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "jaegertracing.io/v1",
+          "kind": "Jaeger",
+          "metadata": {
+            "name": "my-jaeger"
+          },
+          "spec": {
+            "strategy": "allInOne"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: quay.io/jaegertracing/jaeger-operator
```

Comment:
@iblancasa you may need to add a image tag here?

Reply:
Oh! Thanks! It seems I fell into https://github.com/jaegertracing/jaeger-operator/issues/1682